signing/multisig/converters.go

```diff
@@ -0,0 +1,41 @@
+package multisig
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-crypto"
+)
+
+func convertBytesToPubKeys(pubKeys [][]byte, kg crypto.KeyGenerator) ([]crypto.PublicKey, error) {
```

Comment:
should we allow this function to be called with an empty slice of pubKeys bytes?

Reply:
added an error